daffodil-runtime1-unparser/src/main/scala/edu/illinois/ncsa/daffodil/processors/unparsers/IBM4690PackedDecimalUnparsers.scala

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package edu.illinois.ncsa.daffodil.processors.unparsers
+
+import edu.illinois.ncsa.daffodil.processors.parsers.HasKnownLengthInBits
+import edu.illinois.ncsa.daffodil.processors.ElementRuntimeData
+import edu.illinois.ncsa.daffodil.processors.ParseOrUnparseState
+import edu.illinois.ncsa.daffodil.util.DecimalUtils._
+import java.lang.{ Long => JLong }
+import edu.illinois.ncsa.daffodil.processors.parsers.HasRuntimeExplicitLength
+import edu.illinois.ncsa.daffodil.processors.Evaluatable
+import edu.illinois.ncsa.daffodil.schema.annotation.props.gen.LengthUnits
+import edu.illinois.ncsa.daffodil.schema.annotation.props.gen.BinaryNumberCheckPolicy
+
+abstract class IBM4690PackedNumberBaseUnparser(
+  e: ElementRuntimeData)
+  extends PackedBinaryBaseUnparser(e) {
+
+  def fromBigInteger(num: String, nBits: Int, signCodes: String, policy: BinaryNumberCheckPolicy): Array[Byte] = IBM4690FromBigInteger(num, nBits)
```

Comment:
Could IBM4690FromBigInteger and related functions be renamed? The capitalization makes it look like it's a class being allocated rather than a function being called. Functions should almost always started with a lowercase letter.  Maybe just make ibm lowercase? It also might help to keep the DecimalUtils object, e.g. ``... = DecimalUtils.ibm4690FromBigInteger(...)``